src/globus_sdk/experimental/globus_app/globus_app.py

```diff
@@ -144,7 +140,7 @@ class GlobusApp(metaclass=abc.ABCMeta):
 
     def __init__(
         self,
-        app_name: str,
+        app_name: str = "DEFAULT",
```

Comment:
Open Question: Is there a better default than this?

This is the name that will be used by default in (1) token-based file caching (after being 'slugified'), (2) http request headers, and ultimately (3) native login flow prefilled named grants.

My initial version of this was named `"Unnamed Globus App"` (becomes /unamed-globus-app/tokens.json) but I swapped it to `"DEFAULT"`. Curious what opinion reviewers have.

Reply:
I think "Unnamed Globus App" looks good to me. It's longer but includes the keywords necessary for developers who care to figure out where it's coming from.

Reply:
I think I quite like this change, FWIW.

Reply:
> I think I quite like this change, FWIW.

Do you mean the change to name it `"DEFAULT"`, the change to name if `"Unnamed Globus App"` or the decision at large to not require an explicit app name?

Reply:
I couldn't see Kurt's comment when I wrote that cause I was in code view. I like "default", but my opinion is not that strong, I can stand down on it if Kurt feels strongly.

Reply:
I don't feel strongly, but I think `"DEFAULT"` versus `"Unnamed Globus App"` is comparable to raising `Exception` versus `ZeroDivisionError`, insofar as it gives developers a clear indication what they need to be looking for.

Reply:
Hmm, that's a good point.

I'd read this slightly differently, though: since this change makes the app name fully optional, calling it the "default" app is not only accurate, it's also precise. _But_ your point is well taken, and I do see the value of the alternative. It just yields slightly clunkier paths to the credentials, which is a _very modest_ concession.

I'm happy enough with either option, I'll step back from this one and let y'all go with what you prefer!

Reply:
I've left separate feedback that I'm concerned including the app name in the token cache path may not be ideal anyway. If that feedback is accepted, then the clunkiness concern might be fully addressed.

Reply:
I think Kurt's convinced me; I'll rename back to `Unnamed Globus App`.